Dockerfile.image-service-build

```diff
@@ -1,6 +1,20 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.21
+FROM registry.ci.openshift.org/openshift/release:golang-1.21 AS golang
+
 ENV GO111MODULE=on
 ENV GOFLAGS=""
 
-COPY hack/setup_env.sh ./
-RUN ./setup_env.sh
```

Comment:
Can we delete this script or continue to use it here?
It doesn't make sense to me to have two places where we need to maintain these dependency versions.